ansible_collections/arista/cvp/molecule/cv_validate_v3/device_validate_config.yaml

```diff
@@ -0,0 +1,496 @@
+
+    CVP_DEVICES_ERROR_CVP:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        cvp_configlets:
+          - validate_error
+    CVP_DEVICES_ERROR_LOCAL:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        local_configlets:
+          validate_error_local: "ruter bgp 1111\n   neighbor 1.1.1.1 remote-bs 111"
+
+    CVP_DEVICES_VALID_CVP:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        cvp_configlets:
+          - validate_valid
+    CVP_DEVICES_VALID_LOCAL:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        local_configlets:
+          validate_valid_local: "interface Ethernet1\n  description test_validate"
+
+    CVP_DEVICES_MULTILINE_CVP:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        cvp_configlets:
+          - validate_error_multiline
+    CVP_DEVICES_MULTILINE_LOCAL:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        local_configlets:
+          validate_error_multiline_LOCAL: "interface Ethernet10\n  description test\n!\n\nruter bgp 1111\n  neighbor 1.1.1.1 remote-bs 10\n!\ninterface Ethernet1\n  spanning-tree portfast\n!"
+
+    CVP_DEVICES_MULTIPLE_CVP:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        cvp_configlets:
+        - validate_error
+        - validate_valid
+        - validate_warning
+    CVP_DEVICES_MULTIPLE_LOCAL:
+      - device_name: leaf1
+        search_type: hostname #[hostname | serialNumber | fqdn]
+        local_configlets:
+          validate_warning_local: "interface Ethernet1\n   spanning-tree portfast"
+          validate_valid_local: "interface Ethernet1\n  description test_validate"
+          validate_error_local: "ruter bgp 1111\n   neighbor 1.1.1.1 remote-bs 111"
+    CVP_CONFIGLET:
+      validate_warning: "interface Ethernet1\n   spanning-tree portfast"
+      validate_valid: "interface Ethernet1\n  description test_validate"
+      validate_error: "ruter bgp 1111\n   neighbor 1.1.1.1 remote-bs 111"
+      validate_error_multiline: "interface Ethernet10\n  description test\n!\n\nruter bgp 1111\n  neighbor 1.1.1.1 remote-bs 10\n!\ninterface Ethernet1\n  spanning-tree portfast\n!"
+  tasks:
+  - name: "Push configlet"
+    arista.cvp.cv_configlet_v3:
+      configlets: "{{CVP_CONFIGLET}}"
+      state: present
+
+#####################################
+#     WARNING CONFIGLET             #
+#####################################
+
+  - name: Validate CVP warning configlet with stop_on_error
+    arista.cvp.cv_validate_v3:
+      device: "{{CVP_DEVICES_WARNING_CVP}}"
+      validate_mode: stop_on_error
+    register: STOP_ON_ERROR_WARNING_CONFIG_CVP
+  - assert:
+      that:
+        - "STOP_ON_ERROR_WARNING_CONFIG_CVP.changed == true"
```

Comment:
Why is the `changed` value expected to be true? Does the validation module change anything on CVP? I would expect `success` to show whether validation completed without errors/warnings (depending on `validate_mode` ).

Reply:
@noredistribution